modules/hivemq-edge-module-http/src/main/java/com/hivemq/edge/adapters/http/config/HttpAdapterConfig.java

```diff
@@ -94,6 +93,12 @@ public HttpAdapterConfig(
         return id;
     }
 
+    @Override
+    public @NotNull List<String> calculateAllUsedTags() {
+        // TODO
```

Comment:
Track as ticket or address before merging.

Reply:
yeah the method is in place for the check when a tag gets deleted or an adapter gets added.

Reply:
https://hivemq.kanbanize.com/ctrl_board/57/cards/27234/details/